src/containers/my-courses/add-course-with-input/AddCourseWithInput.styles.ts

```diff
@@ -17,6 +17,7 @@ export const styles = {
     maxWidth: '285px',
     border: '1px solid',
     borderColor: 'primary.200',
-    borderRadius: '6px'
+    borderRadius: '6px',
+    py: { xs: '4px', sm: '4px' }
```

Comment:
Remove this style or change the "sort by" filter, we decided to do "sort by" smaller

Reply:
> Remove this style or change the "sort by" filter, we decided to do "sort by" smaller

removed